praw/__init__.py

```diff
@@ -949,6 +950,19 @@ def get_popular_subreddits(self, *args, **kwargs):
         url = self.config['popular_subreddits']
         return self.get_content(url, *args, **kwargs)
 
+    def get_default_subreddits(self, *args, **kwargs):
+        """Return a get_content generator for the default subreddits.
+
+        The additional parameters are passed directly into
+        :meth:`.get_content`. Note: the `url` parameter cannot be altered.
+
+        """
+        url = self.config['default_subreddits']
+        if (len(args) < self.get_content.__code__.co_varnames.index('limit') and
+                'limit' not in kwargs):
+            kwargs['limit'] = 100
```

Comment:
> With that said, do you think it would be better to change the default for all the methods?

Personally I think it is probably a good idea for praw, since having extra data won't hurt. However, reddit probably has this default for some reason(maybe it puts unnecessary load on the servers?). I think the decision is up to you, since you are the maintainer. 

If you are ok with changing the `limit` in `get_content()` to `100`, I'll definitely support this decision. In fact, I actually tried changing it, however a lot of tests failed because they require logging in. I don't think I can easily fix all of them, since they require mod permissions for `reddit_api_test`. Should I just leave them failing for you to fix?